Tests/SwiftSDKGeneratorTests/EndToEndTests.swift

```diff
@@ -319,17 +333,27 @@ func buildTestcases(config: SDKConfiguration) async throws {
       logger, scratchPath: tempDir.path, withArguments: config.sdkGeneratorArguments)
   }
 
-  logger.info("Built SDK")
+  logger.info("Built Swift SDK")
+
+  // Cleanup
+  let cleanupSdk: () async -> Void = {
+    logger.info("Removing Swift SDK to cleanup...")
+    try? await Shell.run("swift experimental-sdk remove \(bundleName)")
+  }
 
   for testcase in testcases {
-    try await FileManager.default.withTemporaryDirectory(logger: logger) { tempDir in
-      try await buildTestcase(logger, testcase: testcase, bundleName: bundleName, tempDir: tempDir)
+    do {
+      try await FileManager.default.withTemporaryDirectory(logger: logger) { tempDir in
+        try await buildTestcase(
+          logger, testcase: testcase, bundleName: bundleName, tempDir: tempDir)
```

Comment:
`swift-format` is super annoying with its unbalanced parenthesis re-formatting 🙄 
```suggestion
          logger, testcase: testcase, bundleName: bundleName, tempDir: tempDir
        )
```